cspell.json

```diff
@@ -0,0 +1 @@
"lables","lastest","ldflags","localhostgate","loggingexporter","logstest","mapconverter","mapstructure","marshalers","mdatagen","mdatagen's","memorylimiter","memorylimiterextension","memorylimiterprocessor","metadatatest","metricfamily","metricreceiver","metricsexporter","metricsgenerationprocessor","metricstransformprocessor","mininum","mostynb","muehle","multiclient","multimod","mycert","myconnector","myexporter","myextension","myorg","myprocessor","myreceiver","myrepo","mysite","nonclobbering","nopexporter","nopreceiver","nosuchprocessor","notls","obsereporttest","obserporttest","obsreceiver","obsreport","obsreporttest","oidcauthextension","oltp","okey","omitempty","omnition","opencensus","opencensusexporter","opencensusreceiver","otelcol","otelcoltest","otelcorecol","otelsvc","oteltest","otelzap","otlpexporter","otlpgrpc","otlphttp","otlphttpexporter","otlphttpexporter's","otlphttpreceiver","otlpjson","otlpmetrics","otlpreceiver","otlptext","overwritepropertiesconverter","overwritepropertiesmapconverter","parserprovider","pcommon","pdata","pdatagen","pdatagrpc","perfcounters","perflib","pipelineprofiles","pipelinez","pjanotti","plog","plogotlp","plogs","pmetric","pmetricotlp","policyevaluation","pprof","pprofextension","pprofile","pprofileotlp","preconfigured","priya","probabilisticsamplerprocessor","processorhelper","processorhelperprofiles","processorprofiles","processortest","processscraper","proctelemetry","prometheusexporter","prometheusreceiver","prometheusremotewrite","prometheusremotewriteexporter","protogen","protos","ptraceotlp","receiverhelper","receiverprofiles","receivertest","repeate","resourcedetection","resourcedetectionprocessor","resourceprocessor","resourcetolabel","retryable","rpcz","rrschulze","safelist","samplereceiver","samplingdecision","samplingprocessor","sarama","sattributes","sattributesprocessor","scrapererror","scraperhelper","scrapertest","semconv","semconvgen","servicetelemetry","servicetest","servicez","sfixed","sharedcomponent","sigstore","someclientid","someclientsecret","somevalue","songy","spanmetricsconnector","spanmetricsprocessor","spanprocessor","sprocessor","statusdata","statusreporting","statuswatcher","stdlib","stencel","stretchr","subcomponent","subcomponents","subpackages","swiatekm","tailsampling","tchannel","telemetrygen","testcomponents","testconverter","testdata","testfunc","testonly","testprovider","testutil","tocstop","tracecontext","traceid","tracesonmemory","tracetranslator","tracez","triagers","triaging","uints","unixgram","unixpacket","unkeyed","unmarshal","unmarshalling","Unmarshalable","unmarshalls","unshallow","unstarted","userfriendly","validatable","vanityurl","vmmetrics","withauth","xconfighttp","xconfmap","xconnector","xconsumer","xconsumererror","xexporter","xexporterhelper","xextension","xpipeline","xprocessor","xprocessorhelper","xreceiver","yamlmapprovider","yamlprovider","yamls","zapcore","zipkin","zipkinexporter","zipkinreceiver","zipkinv","zpages","zpagesextension","zstd"]}
```

Comment:
can this file exist in the .github/workflows directory?

Reply:
Sure, just moved the this folder and changed the workflow config.